test_defopt.py

```diff
@@ -662,6 +687,12 @@ def foo(*, bar):
         '''), globals_)
         self.assertNotIn('default', self._get_help(globals_['foo']))
 
+    def test_namedtuple(self):
+        def main(foo, bar=None):
+            pass
+        main.__annotations__ = {'foo': Pair, 'bar': Pair}
+        self.assertIn('--bar first second', self._get_help(main))
```

Comment:
What does the help look like for a plain tuple? Would like to get a test for that too.